engine/src/rendertext.cpp

```diff
@@ -52,8 +52,8 @@
 
 #ifdef HAVE_SDL
 
-static SDL_mutex *_rtextSDLMutex() {
```

Comment:
I'm betting this (rendertext.*) and command.* can be deleted. I checked and command.h is referenced from main.h, which isn't referenced at all. 

Probably not a good idea to do this as part of this PR.